config/theme.go

```diff
@@ -84,5 +84,45 @@ func GetColor(key string) tcell.Color {
 func GetColorString(key string) string {
 	themeMu.RLock()
 	defer themeMu.RUnlock()
-	return fmt.Sprintf("#%06x", theme[key].TrueColor().Hex())
+	color := theme[key].TrueColor()
+	if color == tcell.ColorDefault {
+		return "-"
+	}
+	return fmt.Sprintf("#%06x", color.Hex())
+}
+
+// GetReadableColor Returns ColorBlack if the color is brighter than gray
+// otherwise returns ColorWhite
+func GetReadableColor(bg tcell.Color) tcell.Color {
+	if bg == tcell.ColorDefault {
+		return tcell.ColorWhite
+	}
+	r, g, b := bg.RGB()
+	luminance := 77*r + 150*g + 29*b + (1<<7)>>8
+	const gray = 0x77 + 0x77 + 0x77 // The sum of gray's R, G, and B
```

Comment:
```suggestion
	const gray = 119 // The middle gray
```

It doesn't need to be added 3 times because `luminance` is not the sum of RGB, it's one value from 0-255 that represents them all.